sys/net/network_layer/ng_ndp/ng_ndp.c

```diff
@@ -28,6 +28,7 @@
 #include "vtimer.h"
 
 #include "net/ng_ndp.h"
+#include "net/ng_fib.h"
```

Comment:
Hmm, providing simply the header without the FIB, i.e. MODULE_FIB, will have no implementation then (leading to unresolved symbols). 
Or am I mix something up?